docs/release-testing.md

```diff
@@ -0,0 +1,11 @@
+With every rancher release, a new operator version may or may not be released. _This document provides a reference to what tests should be run for both the cases._
+
+For release testing, we test against development Rancher builds; for e.g. to test 2.11, we use `latest/devel/2.11` rancher version (`v2.11-head` image tag). These builds are similar to RC builds, as both utilize the `dev-v2.x` branch of the Rancher chart repository, where application charts for operators are prepared for upcoming release.
+
+Before validation, always sync with developers to ensure the desired versions have been merged into the charts and Rancher projects.
+
+`P0` suite must be run to test every new rancher release, preferably from Rancher Dashboard. This suite tests provisioning and importing clusters, k8s upgrade of the control plane and nodes, and CRUD operations of the nodegroup/nodepool.
+
+If the operator contains any update, the [entire test suite](https://app.qase.io/project/HP) must be tested except `K8sChartSupportUpgrade*`; unless the operator also adds support for a new K8s version.
```

Comment:
I see the list of following tests in the automation `Tests to run` description. 
```
p0_provisioning/p0_import
support_matrix_provisioning/support_matrix_import
k8s_chart_support_provisioning/k8s_chart_support_import
p1_provisioning/p1_import
sync_provisioning/sync_import
```

It would be very helpful if you could refer these tests in the documentation.